src/locales/en/messages.json

```diff
@@ -3562,6 +3562,21 @@
   "personalOwnershipSubmitError": {
     "message": "Due to an Enterprise Policy, you are restricted from saving items to your personal vault. Change the Ownership option to an organization and choose from available Collections."
   },
+  "disableSend": {
+    "message": "Disable Send"
+  },
+  "disableSendPolicyDesc": {
+    "message": "Do not allow users to create or edit Bitwarden Sends. Deleting existing Sends is still allowed."
```

Comment:
Let's stick with branding since "Send" is a proper noun and avoid plural "Send" if we can help it. Perhaps:
`"Do not allow users to create or edit a Bitwarden Send. Deleting an existing Send is still allowed."`

If we can't help it, it's fine, but you'll also need to allow variance in pluralization in the language for translation (adding a "s" doesn't work in most languages for making a proper noun plural, and a description of that would be difficult to convey to translators).